doc/mssql_examples_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	mssql "github.com/denisenkom/go-mssqldb"
+)
+
+var (
+	debug         = flag.Bool("debug", false, "enable debugging")
+	password      = flag.String("password", "", "the database password")
+	port     *int = flag.Int("port", 1433, "the database port")
+	server        = flag.String("server", "", "the database server")
+	user          = flag.String("user", "", "the database user")
+	database      = flag.String("database", "", "the database name")
+)
```

Comment:
You will get redefinition errors here if this file is in `package mssql_test`, since these variables are already defined in `newconnector_example_test.go`. You can just remove this whole block of variables.